subset/dot1x/test_dot1x

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+
+source reporting.sh
+
+REPORT=/tmp/report.txt
+TEST_NAME="dot1x"
+TEST_DESCRIPTION="Dot1x authentication tests"
+
+python3 authenticator/test_dot1x.py /tmp/dot1x_result.txt
+
+while [ ! -f /tmp/dot1x_result.txt  ]; do
```

Comment:
what's the purpose of this while loop?

Reply:
Basically to poll to check if the file has been written to. The python script would write the auth success/fail once the tests go through.

Reply:
Changed.